test/templates/bespoke.ts

```diff
@@ -972,6 +972,32 @@ describe("Bespoke template's browser context", () => {
             }))
         })
       })
+
+      describe('Drag resize', () => {
+        it('resizes on drag', () =>
+          testPresenterView(({ deck }) => {
+            Object.defineProperty(document.documentElement, 'clientWidth', {
+              value: 1000,
+            })
```

Comment:
This assigning may make a side effect in future test cases within the same file because `document.documentElement` will not reset between each test cases. Consider to revert the previous value after testing.

```typescript
const { clientWidth } = document.documentElement

try {
  Object.defineProperty(document.documentElement, 'clientWidth', {
    configurable: true,
    value: 1000,
  })

  // ... some of tests ...
} finally {
  Object.defineProperty(document.documentElement, 'clientWidth', {
    configurable: true,
    value: clientWidth,
  })
}
```

Reply:
fixed in b458417e

Reply:
https://github.com/marp-team/marp-cli/commit/b458417e4ab6db0f34ffc2de453875e7e693e737 still makes the side effect in other test cases because `beforeAll` and `afterAll` will execute before and after running test cases in the whole of file.

Reply:
um? my understanding is that the beforeall and afterall is scoped to the contents of the `describe` not the whole file?

Reply:
> um? my understanding is that the beforeall and afterall is scoped to the contents of the `describe` not the whole file?

yeah, just proved this by debugging and looking at `document.documentElement.clientWidth` in another test case.

Reply:
Oops. I missed https://jestjs.io/docs/setup-teardown#scoping.